public/index.html

```diff
@@ -3,9 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>GitProfiler &ndash; Github Stats</title>
+
```

Comment:
How bout also adding a meta description?

```
<meta name="description" content={description} />
```

This will help in SEO, so when people google search your site they'll see the title and description in google searches. Also when you copy and paste the link, it will show the description of the site also.